source/libsass.html.haml

```diff
@@ -37,11 +37,17 @@ title: libSass
   %li#go
     :markdown
       ### Go
-      [Wellington](https://github.com/wellington/wellington) is an extension to libsass that adds spriting and is available on brew:
-
-      `brew install wellington`
-
-      There are also three other libsass wrappers in go: [gosass](https://github.com/moovweb/gosass), [go-sass](https://github.com/SamWhited/go-sass) and [go_sass](https://github.com/suapapa/go_sass) which have not been updated in a while.
+      [go-libsass](https://github.com/wellington/go-libsass) has the most active
```

Comment:
Go-libsass was updated this week and points to master of libsass.

On Wed, Jan 20, 2016, 4:53 PM Michael Mifsud <notifications@github.com>
wrote:

> In source/libsass.html.haml
> <https://github.com/sass/sass-site/pull/157#discussion_r50334232>:
>
> > @@ -37,11 +37,17 @@ title: libSass
> >    %li#go
> >      :markdown
> >        ### Go
> > -      [Wellington](https://github.com/wellington/wellington) is an extension to libsass that adds spriting and is available on brew:
> > -
> > -      `brew install wellington`
> > -
> > -      There are also three other libsass wrappers in go: [gosass](https://github.com/moovweb/gosass), [go-sass](https://github.com/SamWhited/go-sass) and [go_sass](https://github.com/suapapa/go_sass) which have not been updated in a while.
> > +      [go-libsass](https://github.com/wellington/go-libsass) has the most active
>
> The last commit to go-libsass is almost a year ago. There have been
> several major LibSass releases since then. IMHO this project looks to be
> abandoned.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sass/sass-site/pull/157/files#r50334232>.
>